packages/flutter/pubspec.yaml

```
@@ -10,7 +10,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  parse_server_sdk: ^3.1.2
+  parse_server_sdk: ^3.1.15
```

```suggestion
  parse_server_sdk: ^3.1.2
```

This is not necessary since it's already a range